README.md

```diff
@@ -49,7 +52,9 @@
 
 1. Source the `env` file: `. ./env`.
 
-1. Run `./hack/create.sh $RESOURCEGROUP` to deploy a cluster.
+1. Determine an appropriate resource group name for your cluster (e.g. for a test
+   cluster, you could call it `$USER-test`). Then `export RESOURCEGROUP` and run
+   `./hack/create.sh $RESOURCEGROUP` to deploy a cluster.
```

Comment:
personally, I have the RESOURCEGROUP set in my `env` file.  Then I use `make create` and `make delete` to bring up and delete the cluster respectively.